### ISSUE TITLE
🐛 Bug Report: In PythonSDK, for create_deployment() function activate is showing error for True and False. but working for 0 and 1.

### ISSUE THREAD

**Issue by garvitomer** [appwrite/appwrite#3169]
### 👟 Reproduction steps

1. run functions.create_deployement(....., activate=True) in Python SDK

### 👍 Expected behavior

1.  Code should deploy the function on server

### 👎 Actual Behavior

1. Showing Error: Invalid activate: Value must be a valid boolean

### 🎲 Appwrite version

Version 0.13.x

### 💻 Operating system

MacOS

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by Meldiron** [comment]
## Background

I remember we had to stringify the active status in JS for whatever reason... Most likely FormData's body does not support boolean type.

![CleanShot 2022-05-06 at 13 12 44](https://user-images.githubusercontent.com/19310830/167121234-30cf05e7-b9fb-43b4-8bd2-8e9c256a59f0.png)

## Manual QA

- I can confirm the same problem is there for me too

![CleanShot 2022-05-06 at 13 26 27](https://user-images.githubusercontent.com/19310830/167122843-59f9301d-65b9-48c2-805c-5919eb971489.png)

- I can confirm that doing `active='true'` and `activate='false'` solves the problem (notice the lowercase first letter)

![CleanShot 2022-05-06 at 13 26 05](https://user-images.githubusercontent.com/19310830/167122766-770b5ac7-9b06-4a9c-a94a-cd946c4533b7.png)

## Solution

In `appwrite/serivces/functions.py`, inside `create_deployment`, we should do following to convert boolean param into a string:

```py
            params['activate'] = 'true' if activate == True else 'false'
```

## Action

- I created PR to address this problem the same way we did in Node: https://github.com/appwrite/sdk-generator/pull/440
- I made myself a note to try this in all SDKs and see if they have the same problem

**Comment by Meldiron** [comment]
Addressed in 0.14 release. Please re-open issue if you have same problem after upgrading.
